src/style/tokens/base.scss

```diff
@@ -149,6 +149,7 @@
   --page-color-secondary-contrast: var(--typography-color-body);
   --page-color-shadow: rgba(12, 12, 12, 0.3);
   --page-container-size: 1240px;
+  --page-color-background-alt2: #e3dbff;
```

Comment:
I think we don't need a n additional token here, as we can construct a similar color by altering the HSL values found in:

```
  --brand-color-hue: #{hue($brand-color)};
  --brand-color-saturisation: #{saturation($brand-color)};
  --brand-color-lightness: #{lightness($brand-color)};
  ```